### PR TITLE
README: fixing CI badge

### PR DESCRIPTION
**Pull request by valbers**
Just fixing the CI badge 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 F# implementation of Facebook [GraphQL query language specification](https://facebook.github.io/graphql).
 
-[![Build and Test](https://github.com/fsprojects/FSharp.Data.GraphQL/actions/workflows/dotnetcore.yml/badge.svg)](https://github.com/fsprojects/FSharp.Data.GraphQL/actions/workflows/dotnetcore.yml)
+[![Build and Test](https://github.com/fsprojects/FSharp.Data.GraphQL/actions/workflows/publish_ci.yml/badge.svg)](https://github.com/fsprojects/FSharp.Data.GraphQL/actions/workflows/publish_ci.yml)
 
 [![Join the chat at https://gitter.im/FSharp-Data-GraphQL/community](https://badges.gitter.im/FSharp-Data-GraphQL/community.svg)](https://gitter.im/FSharp-Data-GraphQL/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (xperiandri) [inline README.md:8]: Probably we should remove this as no one looks there. What do you think?

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 F# implementation of Facebook [GraphQL query language specification](https://facebook.github.io/graphql).
 
-[![Build and Test](https://github.com/fsprojects/FSharp.Data.GraphQL/actions/workflows/dotnetcore.yml/badge.svg)](https://github.com/fsprojects/FSharp.Data.GraphQL/actions/workflows/dotnetcore.yml)
+[![Build and Test](https://github.com/fsprojects/FSharp.Data.GraphQL/actions/workflows/publish_ci.yml/badge.svg)](https://github.com/fsprojects/FSharp.Data.GraphQL/actions/workflows/publish_ci.yml)
```
Comment (xperiandri) [inline README.md]:
```suggestion
[![Publish to GitHub](https://github.com/fsprojects/FSharp.Data.GraphQL/actions/workflows/publish_ci.yml/badge.svg)](https://github.com/fsprojects/FSharp.Data.GraphQL/actions/workflows/publish_ci.yml)
[![Publish to NuGet](https://github.com/fsprojects/FSharp.Data.GraphQL/actions/workflows/publish_release.yml/badge.svg)](https://github.com/fsprojects/FSharp.Data.GraphQL/actions/workflows/publish_release.yml)
```

File: README.md (hunk 1 above)
Comment (valbers) [reply README.md:8]: Good point, but I wouldn't do it right now. This virtual space seems to still exist, although it seems not to be used anymore.
For example, we could announce in that virtual space that we'll close it in a few days. Then close it and then remove this reference here.

Comment (xperiandri) [reply README.md:8]: OK, let's leave then

**Review by xperiandri — APPROVED** [review]

**Pull request merged**